config.json

```diff
@@ -13,6 +13,9 @@
     "https_cert": "server.crt",
     "https_key": "server.key",
     "https_port": ":443",
+    "ice_servers": ["stun:stun.l.google.com:19302"],
+    "ice_username": "",
+    "ice_credential": "",
```

Comment:
I realize in the other project the blank values were left as documentation, but this project at least has the config values documented in the readme, so I'd say lets just omit these empty values.
```suggestion
```

Reply:
Happy with this in the latest commit?